Vic2ToHoI4/Source/HOI4World/HoI4Country.cpp

```diff
@@ -874,4 +874,71 @@ std::optional<HoI4FocusTree> HoI4::Country::getNationalFocus() const
 	{
 		return nullopt;
 	}
+}
+
+void HoI4::Country::addSphereling(std::string Sphereling)
+{
+	double defaultAutonomy = 0.5;
```

Comment:
Since this value is used many places, try placing it at the top of the file and using it everywhere you reference 0.5.
```constexpr double defaultAutonomy = 0.5;```

Reply:
Or maybe I can drop it altogether, since I have defined in HoI4Relations.h
`double		spherelingAutonomy = 0.5;`
Just return that?

Reply:
Oh yeah, if you already have a definition, use that.

Reply:
Yes, and actually most of it was used by LOG that I no longer need, so it is now removed.